purchase_cancel_remaining_delivery/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Purchase Cancel Remaining Delivery",
+    "version": "14.0.1.0.0",
+    "author": "Camptocamp SA,Odoo Community Association (OCA)",
+    "category": "Purchase",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "depends": ["purchase_stock", "purchase_picking_state"],
```

Comment:
`purchase_stock` is already a dependency of `purchase_picking_state`
But why is the dependency to `purchase_picking_state` needed?
I don't think this is needed, we should keep the dependencies small. 

If someone wants to install `purchase_picking_state` it can be installed.

Reply:
`purchase_picking_state` is used to check if the button for this module needs to be displayed or not.
I could not include it but then I would need to re-implement the same computation in this module.

As that module is small and implements only the functionality that is needed here. It makes more sense to me to use it as dependency.

Reply:
Yes you are right i have not seen the invisible.
Could you please remove purchase_stock?

Reply:
Done